task-bht_bold.psyexp

```diff
@@ -68,18 +68,18 @@
   </Settings>
   <Routines>
     <Routine name="breath_out">
-      <PolygonComponent name="polygon_2" plugin="None">
+      <PolygonComponent name="polygon1" plugin="None">
         <Param val="center" valType="str" updates="constant" name="anchor"/>
         <Param val="rgb" valType="str" updates="constant" name="colorSpace"/>
         <Param val="1" valType="num" updates="constant" name="contrast"/>
         <Param val="False" valType="bool" updates="None" name="disabled"/>
         <Param val="" valType="code" updates="None" name="durationEstim"/>
-        <Param val="1.0000, 0.2941, -1.0000" valType="color" updates="constant" name="fillColor"/>
+        <Param val="1.0000, 1.0000, -1.0000" valType="color" updates="constant" name="fillColor"/>
```

Comment:
why this color change?

Reply:
To differentiate more with the last breath (before it was orange, now it is yellow.)